src/image.go

```diff
@@ -320,9 +372,11 @@ func (ir *ImageRenderer) SavePNG(path string) error {
 		}
 
 		w, h := dc.MeasureString(str)
+		w += (w * float64(ir.runeAdditionalWidth(runes[0])))
```

Comment:
this is done to double the width or not, right?

Reply:
Indeed! I tried to go through and add comments on all points. It was light on comments before so wasn't sure on stance there, but hopefully the additions here are good - happy to tweak more!